flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/DefaultResourceAllocationStrategy.java

```diff
@@ -83,6 +87,24 @@ public DefaultResourceAllocationStrategy(
             boolean evenlySpreadOutSlots,
             Time taskManagerTimeout,
             int redundantTaskManagerNum) {
+        this(
+                totalResourceProfile,
+                numSlotsPerWorker,
+                evenlySpreadOutSlots,
+                taskManagerTimeout,
+                redundantTaskManagerNum,
+                new CPUResource(0.0),
+                MemorySize.ZERO);
+    }
```

Comment:
Should be `@VisibleForTesting`. BTW, I think it's overkill to add a new constructor only for test here.